Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -3463,6 +3467,41 @@ public IWebElement LocateElementByLocators(ObservableList<ElementLocator> Locato
             return null;
         }
 
+        public IWebElement LocateElementByFriendlyLocators(ElementLocator locator, ObservableList<ElementLocator> FriendlyLocators, ElementInfo currentPOMElementInfo)
+        {
+            IWebElement friendlyEle = null;
```

Comment:
add try catch to handle the exception in the this function

Reply:
Added